scheduler/config-k8s.edn

```diff
@@ -49,7 +49,7 @@
               :disallowed-var-names #{"BADVAR"}
               :init-container {:command ["/bin/sh" "-c" "echo sample init container running"]
                                :image "byrnedo/alpine-curl:latest"}
-              :sidecar {:command ["cook-sidecar"]
+              :sidecar {:command ["cook-sidecar" "--file-server-port"]
```

Comment:
Nitpick: we should add a comment that the port will be added by Cook, so the command must be structured to accept the port as the last argument.

Reply:
Done